contessa/runner.py

```diff
@@ -27,26 +28,38 @@ def run(
         check_table: Dict,
         result_table: Dict,  # todo - docs for quality name, maybe defaults..
         context: Optional[Dict] = None,
+        destinations: List[Destination] = None,
```

Comment:
Im not sure about this. If we allow multiple destination, we should ensure consistency. Meaning, if DD send fail, we should rollback DB transaction and vice versa. Which requires more code to handle it.

As I think about this, I would for now allow only 1 destination. Just DD or DB. We(kiwi) will mostly use DB and if we would need, we can just run the same code with DD destination or to have a subscriber on DB level that will send it to DD (which is imho better idea in case you are using DB as destination)

What do you think?